templates/puppetdb-DLO-cleanup.timer.epp

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Daily Timer for puppetdb-DLO-cleanup
+
+[Timer]
+# Daily at 8am
+OnCalendar=*-*-* 08:0:00
```

Comment:
At a minimum, we should support customizing the time of day that this service runs. One could imagine wanting to schedule this for a low traffic time, which may be specific per user. It would also be nice to specific frequency. Some users may want to clean up once a week on Saturday.

Also the comment will no longer be accurate, so you can remove it or replace it with a general comment with a link to documentation on the [systemd timer syntax](https://www.freedesktop.org/software/systemd/man/systemd.time.html)